ipaserver/install/cainstance.py

```diff
@@ -499,7 +500,7 @@ def __spawn_instance(self):
         # Directory server
         config.set("CA", "pki_ds_ldap_port", "389")
         config.set("CA", "pki_ds_password", self.dm_password)
-        config.set("CA", "pki_ds_base_dn", self.basedn)
+        config.set("CA", "pki_ds_base_dn", six.text_type(self.basedn))
```

Comment:
I don't see difference between __str__ implementation in `DNSName` and `DN`, I will investigate OptionParser, maybe it changed in Py3 to enforce strings